From 62093c17f5ce8acaa02ed6a1789027aebd81e5e2 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@st.com>
Date: Fri, 22 Sep 2023 13:54:50 +0200
Subject: [PATCH 348/841] feat(stm32mp2-fdts): add memory node

Add ddr node in stm32mp251.dtsi file, which gives addresses to DDR PHY
and DDR controller.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git b398851c5ee9beb2a91eb27545b0bd87e1183ef9 feat(stm32mp2-fdts): add memory node]
Signed-off-by: Yann Gautier <yann.gautier@st.com>
Change-Id: I719bfd1640a8217ff79e79b5b53845b75421d298
---
 fdts/stm32mp251.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/fdts/stm32mp251.dtsi b/fdts/stm32mp251.dtsi
index 12570a3fa..1ab142b3f 100644
--- a/fdts/stm32mp251.dtsi
+++ b/fdts/stm32mp251.dtsi
@@ -430,6 +430,13 @@
 			reg = <0x44230000 0x10000>;
 		};
 
+		ddr: ddr@48040000 {
+			compatible = "st,stm32mp2-ddr";
+			reg = <0x48040000 0x10000>,
+			      <0x48c00000 0x400000>;
+			status = "okay";
+		};
+
 		fmc: memory-controller@48200000 {
 			#address-cells = <2>;
 			#size-cells = <1>;
-- 
2.47.2

